src/sass/components/_path.scss

```diff
@@ -74,7 +74,8 @@ $color-background-path-complete-hover: $color-gray-6;
     }
   }
 
-  .slds-is-won {
+  .slds-is-won,
+  .slds-is-lost {
```

Comment:
@stacyk I wasn't sure if there's a better way to make these changes? Otherwise our local styles here were overriding the default styles we want for `.slds-is-lost`.

Reply:
@jgerigmeyer I reorganized the css a bit to avoid needing to include `:not` state classes. This is a bit cleaner and also it aligns more with their current path styles: https://github.com/salesforce-ux/design-system/blob/69f18a24a65c4d7d006a6e4aa25db27fcefdf2f6/ui/components/path/base/_index.scss